client/app/bundles/shared/components/Input.scss

```diff
@@ -0,0 +1,54 @@
+@import "base.scss";
+
+.label {
+  font-family: $font-family;
+  padding: 12px 0;
+  letter-spacing: 0.0944444em;
+  text-indent: 4px;
+  text-transform: uppercase;
+  opacity: 0.5;
+}
+
+.labelLight {
+  composes: label;
+  color: #ffffff;
```

Comment:
We should use $background-color here. Feel free to update its value in `app/assets/stylesheets/base/_colors.scss` to `#FFFFFF` :)